src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -1011,12 +1004,19 @@ boolean isExhaustOnDuplicateCandidateEnabled() {
     return rawConfig.rules.exhaustOnDuplicateCandidate;
   }
 
-  Set<String> getCandidateCodeList() {
-    return candidateCodeToNameMap.keySet();
+  Set<String> getCandidateNames() {
+    if (candidateNames == null) {
+      candidateNames = new HashSet<>();
+    }
```

Comment:
nit: why not just initialize this when you declare it?

Reply:
1. Nothing else is initialized when declared
2. This was a request from @HEdingfield to add this: https://github.com/BrightSpots/rcv/pull/639#discussion_r1181277098
https://github.com/BrightSpots/rcv/pull/639#discussion_r1179981350

Reply:
I think either way would probably work fine. Also agree with @artoonie that we should make a pass on all this at some point to make it consistent.